src/mlpack/methods/ann/init_rules/glorot_init.hpp

```diff
@@ -150,8 +150,8 @@ inline void GlorotInitializationType<false>::Initialize(MatType& W,
   if (W.is_empty())
     Log::Fatal << "Cannot initialize an empty matrix." << std::endl;
 
-  double var = 2.0 / double(W.n_rows + W.n_cols);
-  GaussianInitialization normalInit(0.0, var);
+  double stddev = 2.0 / double(W.n_rows + W.n_cols);
+  GaussianInitialization normalInit(0.0, stddev);
```

Comment:
I am pretty sure that only the variable name needs to be changed here, I looked online and it seems that what was intended, but I would like someone else to check as well.

Reply:
Actually I think this should be `std::sqrt(2.0 / double(W.n_rows + W.n_cols))`.  As far as I know (I am relatively confident but not 1000% sure), this comes from Eq. 12 in this paper: http://proceedings.mlr.press/v9/glorot10a/glorot10a.pdf

There they are defining the variance, not the standard deviation, so we should take the square root to get the standard deviation.